emote/utils/model.py

```diff
@@ -0,0 +1,24 @@
+# This file contains codes/text mostly restructured from the following github repository
+# https://github.com/facebookresearch/mbrl-lib
+
+
+from typing import Union
+
+import numpy as np
+import torch
+
+
+def to_tensor(x: Union[torch.Tensor, np.ndarray]):
+    if isinstance(x, torch.Tensor):
+        return x
+    if isinstance(x, np.ndarray):
+        return torch.from_numpy(x)
+    raise ValueError("Input must be torch.Tensor or np.ndarray.")
+
+
+def to_numpy(x: Union[torch.Tensor, np.ndarray]):
+    if isinstance(x, torch.Tensor):
+        return x.detach().to("cpu").numpy()
+    if isinstance(x, np.ndarray):
+        return x
+    raise ValueError("Input must be torch.Tensor or np.ndarray.")
```

Comment:
Are we still needing these?

Reply:
It is still nice to have a function for 'to_numpy()'. However, since it is a single line code, I can also remove the function. This is how it looks like now:
def to_numpy(x: torch.Tensor):
    return x.detach().to("cpu").numpy()

Reply:
Hmm. We can keep it for now and see how it feels. I'm generally more for explicit code, but it's quite verbose.